lab2/src/main/java/lab2/Lab02.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package lab2;
+
+import java.util.Random;
+import javax.print.DocFlavor.STRING;
+import java.util.Arrays;
+
+public class Lab02{
+    public static void main(String[] args){
+
+        // Roll a Dice
+        int[] results = roll(4);
+
+        // Check if an array has duplicate values
+        System.out.println("Dup Exists:" + checkDups(results));
+
+        // Calculate Averages
+        System.out.println("Averages:" + calcAverage(results));
+
+        // Daily average temperatures for Seattle, October 1-28 2017
+        int[][] weeklyMonthTemperatures = {
+            {66, 64, 58, 65, 71, 57, 60},
+            {57, 65, 65, 70, 72, 65, 51},
+            {55, 54, 60, 53, 59, 57, 61},
+            {65, 56, 55, 52, 55, 62, 57}
+
+        };
+        displayLowestAverage(weeklyMonthTemperatures);   
+  };
+
+    /**
+     * Roll Dice uses random number to generate number betwwen 0 and 6
+     * 
+     * @param n
+     *            number of times the dice is rolled
+     * @return Array with values of each dice roll
+     */
+    public static int[] roll(int n){
+
+        int[] results = new int[n];
+       
+
+        for(int i=0; i<n; i++){
+            results[i] = (int) (Math.random()*(n - 1)) + 1;           
+        }
+       System.out.println("Dice Rolls:" + Arrays.toString(results));
+       return(results);
+    }
+
+    /**
+     * Finds if the input array has duplicate values
+     * 
+     * @param results
+     *            Array of dice results
+     * @return true if duplicate exists else false
+     */
+    public static String checkDups(int[] results){
```

Comment:
This method can just return the boolean `dupExists`, instead of the boolean.toString().  Then in your tests, you can use `assertTrue()` and `assertFalse()`.